server/app/process_queue.py

```diff
@@ -0,0 +1,98 @@
+import json
+import uuid
+import time
+import redis
+import boto3
+import config
+import base64
+import logging
+import datetime
+from io import StringIO
+import pandas as pd
+
+redis_server = redis.StrictRedis(
+    host=config.REDIS_HOST, port=config.REDIS_PORT, db=config.REDIS_ID
+)
+
+
+def dataframe_to_csv_on_s3(dataframe, filename):
+    """ Write a dataframe to a CSV on S3 """
+
+    csv_buffer = StringIO()
+    dataframe.to_csv(csv_buffer, sep=",", index=False, header=False)
+
+    s3_resource = boto3.resource("s3")
+    s3_resource.Object(config.S3_BUCKET, filename).put(Body=csv_buffer.getvalue())
+
+
+# TODO: Write this function
```

Comment:
is this a real TODO? or is it already done?

Reply:
This was done, sorry